tests/s3/multipart_upload/test_multipart_getput.py

```diff
@@ -829,3 +829,73 @@ def test_multipart_upload_test_40256(self):
         s3_background_io.stop()
         s3_background_io.cleanup()
         self.log.info("ENDED: Test Simple and Multipart upload of an object")
+
+    @pytest.mark.tags('TEST-40745')
+    @pytest.mark.s3_ops
+    @CTFailOn(error_handler)
+    def test_multipart_upload_test_40745(self):
```

Comment:
Are you skipping checksum verification for all I/Os. We need that for completeness of the tests.

Reply:
I will note it and add during TE phase in next sprint